.github/workflows/.test.yml

```diff
@@ -180,6 +181,7 @@ jobs:
         with:
           directory: ./bin/testreports
           flags: ${{ matrix.codecov_flags }}
+          disable_file_fixes: false
```

Comment:
> Seems "disable_file_fixes" setting didn't change much.

Will look a bit more at it in follow-up because it seems to work fine with unit tests using `gotestsum`.